Ledger logs are giving me problems here.  Can you please add a commit here to fix log handling?
```
$ make all -s PLATFORM=msom DEBUG_BUILD=y MODULAR=n APPDIR=/home/eric/projects/tinker
In file included from ../platform/MCU/rtl872x/inc/platform_config.h:25,
                 from ../hal/src/rtl872x/littlefs/filesystem_impl.h:20,
                 from ../hal/shared/filesystem.h:23,
                 from src/ledger/ledger.h:31,
                 from src/ledger/ledger.cpp:30:
../services/inc/logging.h:273:16: error: specialization of '_LogCategoryWrapper<_LogGlobalCategory>' after instantiation
  273 |         struct _LogCategoryWrapper<_LogGlobalCategory> { \
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/ledger/ledger.cpp:47:1: note: in expansion of macro 'LOG_SOURCE_CATEGORY'
   47 | LOG_SOURCE_CATEGORY("system.ledger");
      | ^~~~~~~~~~~~~~~~~~~
../services/inc/logging.h:273:16: error: redefinition of 'struct _LogCategoryWrapper<_LogGlobalCategory>'
  273 |         struct _LogCategoryWrapper<_LogGlobalCategory> { \
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/ledger/ledger.cpp:47:1: note: in expansion of macro 'LOG_SOURCE_CATEGORY'
   47 | LOG_SOURCE_CATEGORY("system.ledger");
      | ^~~~~~~~~~~~~~~~~~~
../services/inc/logging.h:261:8: note: previous definition of 'struct _LogCategoryWrapper<_LogGlobalCategory>'
  261 | struct _LogCategoryWrapper {
      |        ^~~~~~~~~~~~~~~~~~~
../build/module.mk:281: recipe for target '../build/target/system/platform-35/src/ledger/ledger.o' failed
make[1]: *** [../build/target/system/platform-35/src/ledger/ledger.o] Error 1
../build/recurse.mk:11: recipe for target 'system' failed
make: *** [system] Error 2
```

```diff
diff --git a/system/src/ledger/ledger.cpp b/system/src/ledger/ledger.cpp
index e068bd395..978b44911 100644
--- a/system/src/ledger/ledger.cpp
+++ b/system/src/ledger/ledger.cpp
@@ -23,6 +23,9 @@
 
 #if HAL_PLATFORM_LEDGER
 
+#include "logging.h"
+LOG_SOURCE_CATEGORY("system.ledger");
+
 #include <algorithm>
 #include <cstring>
 #include <cassert>
@@ -44,8 +47,6 @@
 #define PB_LEDGER(_name) particle_cloud_ledger_##_name
 #define PB_INTERNAL(_name) particle_firmware_##_name
 
-LOG_SOURCE_CATEGORY("system.ledger");
-
 namespace particle {
 
 using fs::FsLock;
diff --git a/system/src/ledger/ledger_manager.cpp b/system/src/ledger/ledger_manager.cpp
index 6d80afd86..e585f31d3 100644
--- a/system/src/ledger/ledger_manager.cpp
+++ b/system/src/ledger/ledger_manager.cpp
@@ -23,6 +23,9 @@
 
 #if HAL_PLATFORM_LEDGER
 
+#include "logging.h"
+LOG_SOURCE_CATEGORY("system.ledger");
+
 #include <algorithm>
 #include <mutex>
 #include <cstring>
@@ -55,8 +58,6 @@
 
 static_assert(PB_CLOUD(Response_Result_OK) == 0); // Used by value in the code
 
-LOG_SOURCE_CATEGORY("system.ledger");
-
 namespace particle {
 
 using control::common::EncodedString;
```

--- a/system/src/ledger/ledger.cpp
+++ b/system/src/ledger/ledger.cpp
@@ -23,6 +23,9 @@
 
 #if HAL_PLATFORM_LEDGER
 
+#include "logging.h"
+LOG_SOURCE_CATEGORY("system.ledger");
+
 #include <algorithm>
 #include <cstring>
 #include <cassert>
@@ -43,8 +46,6 @@
 
 #define PB_LEDGER(_name) particle_cloud_ledger_##_name
 #define PB_INTERNAL(_name) particle_firmware_##_name
-
-LOG_SOURCE_CATEGORY("system.ledger");
 
 namespace particle {
 

--- a/system/src/ledger/ledger_manager.cpp
+++ b/system/src/ledger/ledger_manager.cpp
@@ -23,6 +23,9 @@
 
 #if HAL_PLATFORM_LEDGER
 
+#include "logging.h"
+LOG_SOURCE_CATEGORY("system.ledger");
+
 #include <algorithm>
 #include <mutex>
 #include <cstring>
@@ -54,8 +57,6 @@
 #define PB_LEDGER(_name) particle_cloud_ledger_##_name
 
 static_assert(PB_CLOUD(Response_Result_OK) == 0); // Used by value in the code
-
-LOG_SOURCE_CATEGORY("system.ledger");
 
 namespace particle {
 

